_docs/torneos/ab21.md

```diff
@@ -18,21 +18,97 @@ tags:
+| Dretcm             |920 |	1430861	|1320531|		1330160|	1330861	|1530591|	1330791|	1030|				0	|	1230441|	1030490	|1420931|	100960	|100740	|12159321
+| fedeo_Molitalia | 1310840 |	100981|			0|	300001|			1210|		1220910|1330700|		1320651|	1110000|	1230581|	200600|	1320520|			1230150|				1430800|12107958
+| thisisabot |1300|				201060|		1300|			1330900|				101160|				1320940|	1200|			1330840	|		1330900|200000|	100780|			1040|			200621|		1210001		| 7332056
+| flechita_veloz |100700|		1220180|		1120820|					960|		200741|			100671|			200851|		660|			100190|1130090|1330000|		100001|			0|		1120000| 6725878
+| GalagaRebirth |101121|					1230000|	1220000|				1090|		100491|					201050|		1330790|			200650|		1240|			200000|1230000|200000|	200001|	940| 6217387
+| Mark-Lex21 |21300|			650|				1330100|		1190|		1130000|			1180|			790|				1180|				1130000|		1180|				1230|			1210|			1230080|	1180|	 4851284
+| delorean |1430101|	100650|			101041|			400001|			650|				650|			200651|			1030650|		650|		960|				1230001|			950|		100650|				730|	 4598349
+| cualquiera 	|1030000|				1230001|			200000|			890|			1070|		1150|			1040|				1230071|		0|		100830|				1020|				200550|		980|				101110| 4098726
+| Alonso |1170|		1160|				100000|		100000|			820|					1170|	100000|				990|			1330650|				580|		1170|				1130450|	1020000|				1200| 3789374
+| 1p0l1t0 |101090|				0|	1170|				1110570|		1230470|			980|					870|			1031300|		660|	1020|			1160|					940|			1290|			1270| 3482804
+| sospechoso |1300|			1300|		1300|				1300|		1230500|				1300|		1300|				1300|			1021300|			0|			380|	100000|			1031300|				1090|	 3393684
+| dubinator |0|				1300|		930|				180|			1300|			31300|				1300|	1000|					1300|			1310000|			200001|			101121|	1230001|					1120| 2880867
+| Los_Reales_Gs |21300|		1300|					1300|	1300|			1300|				31300|			1300|			1300|				1011300|		100740|		750|				920|			100660|			1330151|	 2604935
+| iguana_dye |100880|	730|					100611|			820|	490|					100761|			1100|			1030280|			100641|			100761|	101101|			700|					1140|			100621| 1740650
+| leroyG5 |1290|			1030930|			1300|	1260|					101210|	1270|			101100|			1290|				1170|		1300|					1130|			1180|	1300|					1180| 1246924
+| TheBotLess |100970					|1100|		100000|		100581|					200100|	0|	0|					200001|			100621|					1080|	200000|				100550|				100650|	860| 1206526
+| Tilin |1300|				1300|				1300|	1190|				1300|				1240|	1300|					1031300|	1300|				1300|				1300|	1300|					1300|			1300| 1048044
+| LaUvaLoca |100151|				1010|			150|		1000|				1060|	150|				1000|				1010|	0|				980|		1090|			1150|				550|			200150|	 309464
+| BlackCat |1300|	1300|				1300|				1300|			1300|		1300				|1300|	1300|			1300|					1300|		1300|		650|			1300|				1300|	 17564
+
+	
+## Finales 18/09/21
+
+|                    Upper brackets |  |Fase 1(VO3) |\||Fase 2(VO3)|\||Fase 3(VO3)|\||Fase 4(VO3)|\||Fase 5(VO5) |\|| GANADOR
```

Comment:
Change to `BO1` and `BO3`, (best of <number>).

Reply:
do it 
fix VO to BO